packages/land/package.json

```diff
@@ -46,6 +46,10 @@
       ]
     }
   },
+  "dependencies": {
+    "@manifoldxyz/royalty-registry-solidity": "^2.0.3",
+    "@sandbox-smart-contracts/dependency-royalty-management": "1.0.2"
+  },
```

Comment:
devdep or dependencies ?
and why do we need @manifoldxyz/royalty-registry-solidity ?

Reply:
resolved

Reply:
`dependency-royalty-management` is needed as Dependecies. 
Removed `@manifoldxyz/royalty-registry-solidity`

Reply:
To use `IRoyaltyManager` interface for Land/PolygonLand Contract and `RoyaltyManager` as mock these dependencies are needed

Reply:
actually all the sol sources packages have to be in `dependencies` even @openzeppelin/contracts
so that when we publish it on npm, the sol source are present

Reply:
still missing this

Reply:
Done